src/NumericalAlgorithms/LinearOperators/Divergence.hpp

```diff
@@ -102,4 +111,25 @@ struct DivCompute : db::add_tag_prefix<div, Tag>, db::ComputeTag {
   using argument_tags =
       tmpl::list<Tag, domain::Tags::Mesh<dim>, InverseJacobianTag>;
 };
+
+/// \ingroup DataBoxTagsGroup
+/// \brief Compute the divergence of a `tnsr::I` (vector)
+///
+/// This tag inherits from `db::add_prefix_tag<Tags::div, Tag>`.
+template <typename Tag, typename InverseJacobianTag>
+struct DivVectorCompute : db::add_tag_prefix<div, Tag>, db::ComputeTag {
+ private:
+  using inv_jac_indices =
+      typename db::const_item_type<InverseJacobianTag>::index_list;
+  static constexpr auto dim = tmpl::back<inv_jac_indices>::dim;
+  static_assert(cpp17::is_same_v<typename tmpl::front<inv_jac_indices>::Frame,
```

Comment:
is the compiler error if Tag::type is not a tnsr::I  clear enough, or should there be a static_assert or Requires?

Reply:
I'm not sure I'm the right person to answer than, since my opinion of "clear" compiler errors is probably a bit skewed... However, we don't have that sort of a `static_assert` in the other compute tags so it would be a bit consistent to have it here :man_shrugging: My only opinion is that I don't love having to use `db::item_type` in tags because it requires the full definition of the tag's type